### PR TITLE
SIITBX 463 s2 evolution 8x

### PR DESCRIPTION
**Pull request by Florian1209**
This evolution includes major updates on the products format with respect to the current baseline (PB 03.01; PSD version 14.6).

The S2 evolution include the update of the jp2 reader to manage the source with more 4 bands.

### CODE REVIEW

**Review by marpet — COMMENTED** [review]
I'm fine with it. But Oana should have a look at it too before merging it

**Review by oanahogoiu — CHANGES_REQUESTED** [review]
The changes seem OK.
Just clean the code (remove commented lines, use debug log level instead of info)

File: snap-jp2-reader/src/main/java/org/esa/snap/jp2/reader/internal/JP2TileOpImage.java
Comment (oanahogoiu) [inline snap-jp2-reader/src/main/java/org/esa/snap/jp2/reader/internal/JP2TileOpImage.java:72]: remove commented code

**Review by oanahogoiu — APPROVED** [review]

**Comment by Florian1209** [comment]
It miss a function to enable openJp2 (without the option) from the the number of jpg2 band (>4).
In this case the jpg2 is necessary.

**Comment by Florian1209** [comment]
@oanahogoiu Could you have a look at the new last commit (concerning the activation of the openJp2Jna)?

**Comment by oanahogoiu** [comment]
@Florian1209 It looks OK the usage of openJP2JNA when intentionnally set in the preferences and when there are more than 4 bands (regardless the preference, but openJP2 available).

**Comment by Florian1209** [comment]
@oanahogoiu  Thank you

**Pull request merged**
